service_catalog/api/serializers/accept_request_serializer.py

```diff
@@ -0,0 +1,30 @@
+from service_catalog.api.serializers.dynamic_survey_serializer import DynamicSurveySerializer
+
+
+class AcceptRequestSerializer(DynamicSurveySerializer):
+    def __init__(self, *args, **kwargs):
+        self.target_request = kwargs.pop('target_request')
+        self.user = kwargs.pop('user')
+        self.read_only_form = kwargs.pop('read_only_form', None)
```

Comment:
bolean by default

Reply:
+ move this into the DynamicSurveySerializer.
and remove the next if self.read_only_form block